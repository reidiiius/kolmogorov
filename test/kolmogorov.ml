
let vessel = __MODULE__ in
  Printf.printf "\n\t%s\n\n" vessel;;

