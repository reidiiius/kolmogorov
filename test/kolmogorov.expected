
            :all    :alloys    :find    :help    :keys    :mars

          :beadgcf   :bfbfb   :cgdae   :eadgbe   :fkbjdn   :piano

	i0	j126w7	j136w7	j167w2	j17	j17w2
	j2	j23	j236	j23k6	j246w3	j25	j256	j25w6
	j26	j26w3	j26w34	j2k34	j2k5	j2k56	j2k56m4	j2k6
	j2k6m5	j2k6w3	j2w3	j3	j346w5	j34k6	j36	j3k5
	j3k56m4	j3k5m4	j3k6	j5	j56	j56w7	j5w6	j6
	k1	k12	k125	k12j5	k135m4	k15	k157m6	k17j5
	k1j5	k1j56w7	k1j6	k1j6w7	k2	k235m4	k25	k256
	k257m1	k25m1	k25m17	k26	k26m5	k2j17	k2j5	k2j56
	k2j56w7	k2j5m1	k2j5w6	k2j6	k2m1	k34	k345m2	k34m2
	k5	k56	k56m4	k6	k6m5	n0	n167	n167m4
	n25m6	n26w5	n345	n345w7	n45w2	n5w2	n67m2	n6m2

	head	tail	last

	i0	j126w7	j136w7	j167w2	j17	j17w2
	j2	j23	j236	j23k6	j246w3	j25	j256	j25w6
	j26	j26w3	j26w34	j2k34	j2k5	j2k56	j2k56m4	j2k6
	j2k6m5	j2k6w3	j2w3	j3	j346w5	j34k6	j36	j3k5
	j3k56m4	j3k5m4	j3k6	j5	j56	j56w7	j5w6	j6
	k1	k12	k125	k12j5	k135m4	k15	k157m6	k17j5
	k1j5	k1j56w7	k1j6	k1j6w7	k2	k235m4	k25	k256
	k257m1	k25m1	k25m17	k26	k26m5	k2j17	k2j5	k2j56
	k2j56w7	k2j5m1	k2j5w6	k2j6	k2m1	k34	k345m2	k34m2
	k5	k56	k56m4	k6	k6m5	n0	n167	n167m4
	n25m6	n26w5	n345	n345w7	n45w2	n5w2	n67m2	n6m2


	k1	k12	k125	k12j5	k135m4
	k15	k157m6	k17j5	k1j5	k1j56w7	k1j6	k1j6w7	k2
	k235m4	k25	k256	k257m1	k25m1	k25m17	k26	k26m5
	k2j17	k2j5	k2j56	k2j56w7	k2j5m1	k2j5w6	k2j6	k2m1
	k34	k345m2	k34m2	k5	k56	k56m4	k6	k6m5

	n0	n167	n167m4
	n25m6	n26w5	n345	n345w7	n45w2	n5w2	n67m2	n6m2

	j126w7	j136w7	j167w2	j17	j17w2
	j2	j23	j236	j23k6	j246w3	j25	j256	j25w6
	j26	j26w3	j26w34	j2k34	j2k5	j2k56	j2k56m4	j2k6
	j2k6m5	j2k6w3	j2w3	j3	j346w5	j34k6	j36	j3k5
	j3k56m4	j3k5m4	j3k6	j5	j56	j56w7	j5w6	j6


	k157m6	k1j6	k235m4	k256	k26
	k26m5	k2j17	k2j6	k56	k56m4	k6	k6m5	

	n0	n167m4	n26w5	n345	n45w2	n5w2	

	j136w7
	j167w2	j17	j17w2	j2	j23	j236	j23k6	j246w3
	j26	j26w3	j26w34	j2k56	j2k56m4	j2k6	j2k6m5	j2k6w3
	j2w3	j3	j34k6	j36	j3k56m4	j3k5m4	j3k6	j6


	AgAu
	AgHg	AgMn	AgPb	AgTi	AgUr	AuAg	AuAu	AuHg
	AuNp	AuPb	AuSn	AuUr	CuFe	CuHg	CuNp	CuPb
	CuTi	CuUr	FeCu	FeFe	FeMn	FeNp	FePb	FePu
	FeTi	FeUr	HgAg	HgAu	HgCu	HgHg	HgMn	HgSn
	HgTi	MnAg	MnFe	MnHg	NpAu	NpCu	NpFe	NpSn
	PbAg	PbAu	PbCu	PbFe	PbPb	PbSn	PbTi	PuFe
	SnAu	SnHg	SnNp	SnPb	SnSn	SnTi	TiAg	TiCu
	TiFe	TiHg	TiPb	TiSn	UrAg	UrAu	UrCu	UrFe


	j2	k2	j23	j26	k12	k25	j236	j2k6
	j2w3	j3k5	k125	k1j6	k2j5	k2m1	n167	n345
	j23k6	j26w3	j2k56	k12j5	k25m1	k2j56	j246w3	j26w34
	j2k6m5	j2k6w3	j346w5	k157m6	k235m4	k257m1	k25m17	k2j5m1
	k2j5w6	j2k56m4	k2j56w7

	n167m4 { AgTi AuHg FeNp HgAu NpFe PbSn UrCu }

	n345w7 { AuHg CuUr FeNp HgAu NpFe SnPb TiAg }

	k235m4 { AgMn AuPb FePu HgHg NpCu PbAu UrAg }

	j126w7 { AgUr AuPb CuNp HgHg MnAg PbAu PuFe }

	j6	k5	j36	j56	k15	j56w7	k56m4	j136w7
	j167w2	j3k5m4	k1j6w7	k345m2	n167m4	n345w7

	j2	k2	j23	j26	k12	k25	j236	j2k6
	j2w3	j3k5	k125	k1j6	k2j5	k2m1	n167	n345
	j23k6	j26w3	k12j5	k25m1	k2j56	j126w7	j246w3	j26w34
	j2k6w3	j346w5	k235m4	k2j5m1	k2j5w6	j2k56m4

	Elapsed: 0.000ms runabout_geoffroy


          :beadgcf   :bfbfb   :cgdae   :eadgbe   :fkbjdn   :piano
	HgCu ____ SnSn ____ CuHg PbFe ____ AuAg ____ AgAu ____ FePb 
	____ AgAu ____ FePb HgCu ____ SnSn ____ CuHg PbFe ____ AuAg ____
	____ CuHg PbFe ____ AuAg ____ AgAu ____ FePb HgCu ____ SnSn ____
	____ FePb HgCu ____ SnSn ____ CuHg PbFe ____ AuAg ____ AgAu ____
	PbFe ____ AuAg ____ AgAu ____ FePb HgCu ____ SnSn ____ CuHg PbFe
	HgCu ____ SnSn ____ CuHg PbFe ____ AuAg ____ AgAu ____ FePb HgCu
	AuAg ____ AgAu ____ FePb HgCu ____ SnSn ____ CuHg PbFe ____ AuAg
	SnSn ____ CuHg PbFe ____ AuAg ____ AgAu ____ FePb HgCu ____ SnSn
	AgAu ____ FePb HgCu ____ SnSn ____ CuHg PbFe ____ AuAg ____ AgAu
	CuHg PbFe ____ AuAg ____ AgAu ____ FePb HgCu ____ SnSn ____ CuHg
	FePb HgCu ____ SnSn ____ CuHg PbFe ____ AuAg ____ AgAu ____ FePb
	____ AuAg ____ AgAu ____ FePb HgCu ____ SnSn ____ CuHg PbFe ____
	____ SnSn ____ CuHg PbFe ____ AuAg ____ AgAu ____ FePb HgCu ____
	n0-beadgcf-3971809718987134967
	PbFe ____ AuAg ____ AgAu ____ FePb HgCu ____ SnSn ____ CuHg PbFe
	HgCu ____ SnSn ____ CuHg PbFe ____ AuAg ____ AgAu ____ FePb HgCu
	AuAg ____ AgAu ____ FePb HgCu ____ SnSn ____ CuHg PbFe ____ AuAg
	SnSn ____ CuHg PbFe ____ AuAg ____ AgAu ____ FePb HgCu ____ SnSn
	AgAu ____ FePb HgCu ____ SnSn ____ CuHg PbFe ____ AuAg ____ AgAu
	CuHg PbFe ____ AuAg ____ AgAu ____ FePb HgCu ____ SnSn ____ CuHg
	FePb HgCu ____ SnSn ____ CuHg PbFe ____ AuAg ____ AgAu ____ FePb
	n0-bfbfb-3971809718987134967
	FePb HgCu ____ SnSn ____ CuHg PbFe ____ AuAg ____ AgAu ____ FePb
	PbFe ____ AuAg ____ AgAu ____ FePb HgCu ____ SnSn ____ CuHg PbFe
	FePb HgCu ____ SnSn ____ CuHg PbFe ____ AuAg ____ AgAu ____ FePb
	PbFe ____ AuAg ____ AgAu ____ FePb HgCu ____ SnSn ____ CuHg PbFe
	FePb HgCu ____ SnSn ____ CuHg PbFe ____ AuAg ____ AgAu ____ FePb
	n0-cgdae-3971809718987134967
	CuHg PbFe ____ AuAg ____ AgAu ____ FePb HgCu ____ SnSn ____ CuHg
	AgAu ____ FePb HgCu ____ SnSn ____ CuHg PbFe ____ AuAg ____ AgAu
	SnSn ____ CuHg PbFe ____ AuAg ____ AgAu ____ FePb HgCu ____ SnSn
	AuAg ____ AgAu ____ FePb HgCu ____ SnSn ____ CuHg PbFe ____ AuAg
	HgCu ____ SnSn ____ CuHg PbFe ____ AuAg ____ AgAu ____ FePb HgCu
	n0-eadgbe-3971809718987134967
	CuHg PbFe ____ AuAg ____ AgAu ____ FePb HgCu ____ SnSn ____ CuHg
	FePb HgCu ____ SnSn ____ CuHg PbFe ____ AuAg ____ AgAu ____ FePb
	AuAg ____ AgAu ____ FePb HgCu ____ SnSn ____ CuHg PbFe ____ AuAg
	SnSn ____ CuHg PbFe ____ AuAg ____ AgAu ____ FePb HgCu ____ SnSn
	AgAu ____ FePb HgCu ____ SnSn ____ CuHg PbFe ____ AuAg ____ AgAu
	CuHg PbFe ____ AuAg ____ AgAu ____ FePb HgCu ____ SnSn ____ CuHg
	n0-fkbjdn-3971809718987134967
	SnSn ____ CuHg PbFe ____ AuAg ____ AgAu ____ FePb HgCu ____ SnSn
	____ FePb HgCu ____ SnSn ____ CuHg PbFe ____ AuAg ____ AgAu ____
	____ AuAg ____ AgAu ____ FePb HgCu ____ SnSn ____ CuHg PbFe ____
	SnSn ____ CuHg PbFe ____ AuAg ____ AgAu ____ FePb HgCu ____ SnSn
	____ FePb HgCu ____ SnSn ____ CuHg PbFe ____ AuAg ____ AgAu ____
	____ AuAg ____ AgAu ____ FePb HgCu ____ SnSn ____ CuHg PbFe ____
	n0-piano-3971809718987134967
	HgCu ____ SnSn ____ CuHg PbFe ____ AuAg ____ AgAu ____ FePb HgCu

	n0-beadgcf-3971809718987134967
	PbFe ____ AuAg ____ AgAu ____ FePb HgCu ____ SnSn ____ CuHg PbFe
	HgCu ____ SnSn ____ CuHg PbFe ____ AuAg ____ AgAu ____ FePb HgCu
	AuAg ____ AgAu ____ FePb HgCu ____ SnSn ____ CuHg PbFe ____ AuAg
	SnSn ____ CuHg PbFe ____ AuAg ____ AgAu ____ FePb HgCu ____ SnSn
	AgAu ____ FePb HgCu ____ SnSn ____ CuHg PbFe ____ AuAg ____ AgAu
	CuHg PbFe ____ AuAg ____ AgAu ____ FePb HgCu ____ SnSn ____ CuHg
	FePb HgCu ____ SnSn ____ CuHg PbFe ____ AuAg ____ AgAu ____ FePb

	i0-piano-3971809718987134967
	____ ____ ____ ____ ____ ____ ____ ____ ____ ____ ____ ____ ____

	k9 ?

	j3-fkbjdn-3971809718987134967
	SnHg UrFe ____ PbAg ____ AuAu ____ AgPb ____ FeUr HgSn ____ SnHg
	____ FeUr HgSn ____ SnHg UrFe ____ PbAg ____ AuAu ____ AgPb ____
	____ AuAu ____ AgPb ____ FeUr HgSn ____ SnHg UrFe ____ PbAg ____
	SnHg UrFe ____ PbAg ____ AuAu ____ AgPb ____ FeUr HgSn ____ SnHg
	____ FeUr HgSn ____ SnHg UrFe ____ PbAg ____ AuAu ____ AgPb ____
	____ AuAu ____ AgPb ____ FeUr HgSn ____ SnHg UrFe ____ PbAg ____

	k1-fkbjdn-3971809718987134967
	HgSn ____ SnHg UrFe ____ PbAg ____ AuAu ____ AgPb ____ FeUr HgSn
	____ AgPb ____ FeUr HgSn ____ SnHg UrFe ____ PbAg ____ AuAu ____
	____ PbAg ____ AuAu ____ AgPb ____ FeUr HgSn ____ SnHg UrFe ____
	HgSn ____ SnHg UrFe ____ PbAg ____ AuAu ____ AgPb ____ FeUr HgSn
	____ AgPb ____ FeUr HgSn ____ SnHg UrFe ____ PbAg ____ AuAu ____
	____ PbAg ____ AuAu ____ AgPb ____ FeUr HgSn ____ SnHg UrFe ____


	k6-cgdae-3971809718987134967
	CuPb PbCu ____ AuSn ____ ____ TiFe FeTi HgAg ____ SnAu ____ CuPb
	____ TiFe FeTi HgAg ____ SnAu ____ CuPb PbCu ____ AuSn ____ ____
	SnAu ____ CuPb PbCu ____ AuSn ____ ____ TiFe FeTi HgAg ____ SnAu
	AuSn ____ ____ TiFe FeTi HgAg ____ SnAu ____ CuPb PbCu ____ AuSn
	HgAg ____ SnAu ____ CuPb PbCu ____ AuSn ____ ____ TiFe FeTi HgAg

	j5-cgdae-3971809718987134967
	AgHg TiFe FeTi ____ ____ SnAu ____ CuPb PbCu ____ AuSn ____ AgHg
	SnAu ____ CuPb PbCu ____ AuSn ____ AgHg TiFe FeTi ____ ____ SnAu
	AuSn ____ AgHg TiFe FeTi ____ ____ SnAu ____ CuPb PbCu ____ AuSn
	____ ____ SnAu ____ CuPb PbCu ____ AuSn ____ AgHg TiFe FeTi ____
	PbCu ____ AuSn ____ AgHg TiFe FeTi ____ ____ SnAu ____ CuPb PbCu

	Elapsed: 0.000ms runabout_jacquard


            :all    :alloys    :find    :help    :keys    :mars

	kolmogorov.ml :help

	kolmogorov.ml :keys

	kolmogorov.ml :mars

	kolmogorov.ml n0 j3

	kolmogorov.ml n0 j3 :cgdae

	kolmogorov.ml :alloys

	kolmogorov.ml :find FeNp FePu

	kolmogorov.ml :find j6 k2

	kolmogorov.ml :all | sensible-pager

	kolmogorov.ml :all :cgdae | sensible-pager


	kolmogorov.ml :help

	kolmogorov.ml :keys

	kolmogorov.ml :mars

	kolmogorov.ml n0 j3

	kolmogorov.ml n0 j3 :cgdae

	kolmogorov.ml :alloys

	kolmogorov.ml :find FeNp FePu

	kolmogorov.ml :find j6 k2

	kolmogorov.ml :all | sensible-pager

	kolmogorov.ml :all :cgdae | sensible-pager


	kolmogorov.ml :help

	kolmogorov.ml :keys

	kolmogorov.ml :mars

	kolmogorov.ml n0 j3

	kolmogorov.ml n0 j3 :cgdae

	kolmogorov.ml :alloys

	kolmogorov.ml :find FeNp FePu

	kolmogorov.ml :find j6 k2

	kolmogorov.ml :all | sensible-pager

	kolmogorov.ml :all :cgdae | sensible-pager


            :all    :alloys    :find    :help    :keys    :mars

          :beadgcf   :bfbfb   :cgdae   :eadgbe   :fkbjdn   :piano

	k1	k12	k125	k12j5	k135m4
	k15	k157m6	k17j5	k1j5	k1j56w7	k1j6	k1j6w7	k2
	k235m4	k25	k256	k257m1	k25m1	k25m17	k26	k26m5
	k2j17	k2j5	k2j56	k2j56w7	k2j5m1	k2j5w6	k2j6	k2m1
	k34	k345m2	k34m2	k5	k56	k56m4	k6	k6m5

	n0	n167	n167m4
	n25m6	n26w5	n345	n345w7	n45w2	n5w2	n67m2	n6m2

	j126w7	j136w7	j167w2	j17	j17w2
	j2	j23	j236	j23k6	j246w3	j25	j256	j25w6
	j26	j26w3	j26w34	j2k34	j2k5	j2k56	j2k56m4	j2k6
	j2k6m5	j2k6w3	j2w3	j3	j346w5	j34k6	j36	j3k5
	j3k56m4	j3k5m4	j3k6	j5	j56	j56w7	j5w6	j6


            :all    :alloys    :find    :help    :keys    :mars

          :beadgcf   :bfbfb   :cgdae   :eadgbe   :fkbjdn   :piano

	k157m6	k1j6	k235m4	k256	k26
	k26m5	k2j17	k2j6	k56	k56m4	k6	k6m5	

	n0	n167m4	n26w5	n345	n45w2	n5w2	

	j136w7
	j167w2	j17	j17w2	j2	j23	j236	j23k6	j246w3
	j26	j26w3	j26w34	j2k56	j2k56m4	j2k6	j2k6m5	j2k6w3
	j2w3	j3	j34k6	j36	j3k56m4	j3k5m4	j3k6	j6


            :all    :alloys    :find    :help    :keys    :mars

          :beadgcf   :bfbfb   :cgdae   :eadgbe   :fkbjdn   :piano

	i0	j126w7	j136w7	j167w2	j17	j17w2
	j2	j23	j236	j23k6	j246w3	j25	j256	j25w6
	j26	j26w3	j26w34	j2k34	j2k5	j2k56	j2k56m4	j2k6
	j2k6m5	j2k6w3	j2w3	j3	j346w5	j34k6	j36	j3k5
	j3k56m4	j3k5m4	j3k6	j5	j56	j56w7	j5w6	j6
	k1	k12	k125	k12j5	k135m4	k15	k157m6	k17j5
	k1j5	k1j56w7	k1j6	k1j6w7	k2	k235m4	k25	k256
	k257m1	k25m1	k25m17	k26	k26m5	k2j17	k2j5	k2j56
	k2j56w7	k2j5m1	k2j5w6	k2j6	k2m1	k34	k345m2	k34m2
	k5	k56	k56m4	k6	k6m5	n0	n167	n167m4
	n25m6	n26w5	n345	n345w7	n45w2	n5w2	n67m2	n6m2

	Elapsed: 0.000ms runabout_ministry

	-------------------------------------------------------------

	     Elapsed: 0.000ms, Failed: 0, Passed: 54, Tested: 54

